optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -12,40 +17,108 @@ const PreferentialTrial: FC<{
-    <Box sx={{ width: trialWidth, padding: theme.spacing(2, 2, 0, 2) }}>
-      <Typography
-        variant="h4"
-        sx={{
-          marginBottom: theme.spacing(2),
-          fontWeight: theme.typography.fontWeightBold,
-        }}
-      >
-        Trial {trial.number} (trial_id={trial.trial_id})
-      </Typography>
+    <Box
+      sx={{
+        width: trialWidth,
+        minHeight: trialHeight,
+        position: "relative",
+      }}
+    >
       <Button
-        variant="outlined"
         onClick={() => {
           hideTrial()
           const best_trials = studyDetail.best_trials
             .map((t) => t.number)
             .filter((t) => t !== trial.number)
           action.updatePreference(trial.study_id, best_trials, [trial.number])
         }}
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+        sx={{
+          "::before": {
+            content: '""',
+            position: "absolute",
+            top: 0,
+            left: 0,
+            width: "100%",
+            height: "100%",
+            backgroundColor: theme.palette.mode === "dark" ? "white" : "black",
+            opacity: 0,
+            zIndex: 1,
+            transition: "opacity 0.3s ease-out",
+          },
+          ":hover::before": {
+            opacity: 0.1,
+          },
+        }}
       >
-        Worst
+        <Box
+          sx={{
+            padding: theme.spacing(2, 2, 0, 2),
+            position: "relative",
+          }}
+        >
+          <TrialNote
+            studyId={trial.study_id}
+            trialId={trial.trial_id}
+            latestNote={trial.note}
+            cardSx={{ marginBottom: theme.spacing(2) }}
+            editable={false}
+          />
+        </Box>
+
+        <ClearIcon
+          style={{
+            opacity: hover ? 0.5 : 0,
+            transition: "opacity 0.3s ease-out",
+          }}
+          sx={{
+            position: "absolute",
+            top: "50%",
+            left: "50%",
+            transform: "translate(-50%, -50%)",
+            color: "red",
+            fontSize: trialWidth / 2,
+          }}
+        />
       </Button>
-      <TrialNote
-        studyId={trial.study_id}
-        trialId={trial.trial_id}
-        latestNote={trial.note}
-        cardSx={{ marginBottom: theme.spacing(2) }}
-      />
+      <IconButton onClick={() => setDetailShown(true)}>
+        <OpenInFullIcon />
+      </IconButton>
+      <Modal open={detailShown} onClose={() => setDetailShown(false)}>
+        <Box
+          sx={{
+            position: "relative",
+            width: "70%",
+            left: "15%",
+            top: theme.spacing(8),
+            backgroundColor: theme.palette.mode === "dark" ? "black" : "white",
+          }}
+        >
+          <TrialListDetail
+            trial={trial}
+            isBestTrial={() => false}
+            directions={[]}
+            objectiveNames={[]}
+          />
+        </Box>
+      </Modal>
```

Comment:
Could you update the styles as the following PR sets `borderRadius` and uses `position: "absolute"`?
https://github.com/optuna/optuna-dashboard/pull/552